x-pack/plugins/security_solution/public/flyout/index.tsx

```diff
@@ -140,6 +140,12 @@ const expandableFlyoutDocumentsPanels: ExpandableFlyoutProps['registeredPanels']
     key: NetworkPanelKey,
     component: (props) => <NetworkPanel {...(props as NetworkExpandableFlyoutProps).params} />,
   },
+  {
+    key: NetworkPreviewPanelKey,
```

Comment:
I think resetting would introduce more complexity. Say we reset history when user closes the flyout. What about if an user has a flyout open, then open another flyout from alerts table? 
- If we are only using history for inter-flyout navigation, we should clear history and start fresh, right? This will be harder to do because currently we don't check whether a flyout is opened within a flyout or on its own. 
- But if we don't start fresh, an user can build a long list by opening many alerts from the table, and accidently closing a flyout will wipe all the history.

I've actually grown to like having all the history. I think with removing duplicates and reordering, it won't grow as quickly. But yeah open to discussing live and walk it through!